fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -634,8 +634,7 @@ public static int analyzeSchemaVersion(Map<String, String> properties) throws An
 
     public static Boolean analyzeUseFastSchemaEvolution(Map<String, String> properties) throws AnalysisException {
         if (properties == null || properties.isEmpty()) {
-            return RunMode.isSharedNothingMode() ? Config.enable_fast_schema_evolution
-                    : Config.experimental_enable_fast_schema_evolution_in_shared_data;
+            return Config.enable_fast_schema_evolution;
```

Comment:
this can be inconsistent with the `buildUseFastSchemaEvolution` logic?

Reply:
> I thought the behavior is, if enable_fast_schema_evolution = false. no matter the property is given true or false, the fast schema evolution should be disabled.

I agree with you, but this goes against the current shared nothing design, so maybe changing that logic in this PR isn't a good idea?